### DISCUSSION TITLE
igniter允许局域网访问

### DISCUSSION THREAD

**Discussion by jonepengcn** [trojan-gfw/igniter#378] [Q&A]
开启局域网访问后，其他开启个人热点就行了吗？状态栏也没用显示目前监听端口，其他设备上应如何配置呢？

**Comment by storytellerF** [comment]
安装igniter的设备开启热点，其他设备连接
其他设备需要安装能够添加socks连接的应用，端口在会在igniter的通知中告知

**Comment by wongsyrone** [comment]
Root过的设备可以使用诸如 VPN Hotspot (https://play.google.com/store/apps/details?id=be.mygod.vpnhotspot) 工具来分享VPN给其他设备，未Root过的据我所知受限于系统，不可以分享VPN链接。
